tests/cli/test_build_and_deploy_aws.sh

```diff
@@ -38,7 +38,7 @@ rlJournalStart
             rlAssertRpm python3-pip
         fi
 
-        rlRun -t -c "pip3 install awscli"
+        rlRun -t -c "pip3 install awscli ansible[aws]"
```

Comment:
```
STDOUT: TASK [Make sure bucket exists] ***********************************************************************************************************
STDOUT: fatal: [localhost]: FAILED! => {"changed": false, "msg": "Python modules \"botocore\" or \"boto3\" are missing, please install both"}
```

Reply:
Installing python3-boto3 from RPM fixed it for me. (Or if you prefer from pypi).

Reply:
Fixed, python3-boto3 rpm is now installed